apollo-portal/src/test/java/com/ctrip/framework/apollo/portal/environment/EnvTest.java

```diff
@@ -97,4 +97,71 @@ public void getName() {
         Env.addEnvironment(name);
         assertEquals(name.trim().toUpperCase(), Env.valueOf(name).toString());
     }
-}
\ No newline at end of file
+
+    @Test
+    public void transformEnvBlankTest() {
```

Comment:
change method name from transformEnvBlankTest to testTransformEnvBlank ?

i.e move the suffix `Test` to prefix `test`.